bundles/org.openhab.io.homekit/src/main/feature/feature.xml

```diff
@@ -10,10 +10,9 @@
         <bundle dependency="true">mvn:javax.json/javax.json-api/1.0</bundle>
         <bundle dependency="true">wrap:mvn:io.netty/netty-all/4.0.32.Final</bundle>
         <bundle dependency="true">wrap:mvn:com.nimbusds/srp6a/1.5.2</bundle>
-        <bundle dependency="true">mvn:net.i2p.crypto/eddsa/0.1.0</bundle>
+        <bundle dependency="true">wrap:mvn:net.vrallev.ecc/ecc-25519-java/1.0.1</bundle>
```

Comment:
Can you also add some wrap URL parameters so these bundles show up nicely in a `bundle:list`? 

E.g.:

```xml
        <bundle dependency="true">wrap:mvn:io.netty/netty-all/4.0.32.Final$Bundle-Name=Netty%20All&amp;Bundle-SymbolicName=io.netty.netty-all&amp;Bundle-Version=4.0.32</bundle>
        <bundle dependency="true">wrap:mvn:com.nimbusds/srp6a/1.5.2$Bundle-Name=Nimbus%20SRP6a&amp;Bundle-SymbolicName=com.nimbusds.srp6a&amp;Bundle-Version=1.5.2</bundle>
        <bundle dependency="true">wrap:mvn:net.vrallev.ecc/ecc-25519-java/1.0.1$Bundle-Name=ECC-25519&amp;Bundle-SymbolicName=net.vrallev.ecc.ecc-25519-java&amp;Bundle-Version=1.0.1</bundle>
        <bundle dependency="true">wrap:mvn:org.zeromq/curve25519-java/0.1.0$Bundle-Name=curve25519-java&amp;Bundle-SymbolicName=org.zeromq.curve25519-java&amp;Bundle-Version=0.1.0</bundle>
```

Reply:
Sure.